src/extensibility/ExtensionManager.js

```diff
@@ -822,9 +794,8 @@ define(function (require, exports, module) {
     exports.updateExtensions        = updateExtensions;
     exports.getAvailableUpdates     = getAvailableUpdates;
     exports.cleanAvailableUpdates   = cleanAvailableUpdates;
-
-    exports.registryObject          = registryObject;
-
+    exports.isRegistryObjectUpdated = isRegistryObjectUpdated;
```

Comment:
I think you can just set `exports.isRegistryObjectUpdated` to false and remove `isRegistryObjectUpdated` 

or you could make `isRegistryObjectUpdated` a method and then keep a private vairable you update in `downloadRegistry` and return true from `isRegistryObjectUpdated` when that variable is set.